### PR TITLE
[Drop-In]Restructure package

### PR DESCRIPTION
**Pull request by abhishek1508**
### Description

The PR restructures the Drop-In UI package into verticals. 

**Before**

drop-In UI
    |__ binder
    |__ coordinator
    |__ components
    |__ internal
    |__ utils

**After**

drop-In UI
    |_ actionbutton
    |_ analytics
    |_ arrival
    |_ backpress
    |_ camera
    |_ infopanel
    |_ internal
    |_ maneuver
    |_ map
    |_ navigationview
    |_ permissions
    |_ roadname
    |_ speedlimit
    |_ tripprogress
    |_ tripsession
    |_ util

### CODE REVIEW

**Comment by RingerJK** [comment]
seems that it's breaking API 🤔

**Comment by LukasPaczos** [comment]
This definitely needs a changelog entry.

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/internal/extensions/NavigationViewContextEx.kt [hunk 1]
```diff
@@ -6,14 +6,14 @@ import android.view.ViewGroup
 import androidx.appcompat.widget.AppCompatTextView
```
Comment (Zayankovsky) [inline libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/internal/extensions/NavigationViewContextEx.kt:6]: You forgot to update package name in this file.

**Comment by abhishek1508** [comment]
> seems that it's breaking API 🤔 

Dtop-In UI is still experimental and we have been breaking APIs since the beginning. This is the right opportunity for these kind of changes as we won't be able to make them after we go stable.

> This definitely needs a changeling entry

I will add one.

Can one of help me get an approval on this today as it's nearly impossible for me to rebase this branch daily?

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/internal/extensions/NavigationViewContextEx.kt (hunk 1 above)
Comment (abhishek1508) [reply libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/internal/extensions/NavigationViewContextEx.kt:6]: Good catch. Fixed.

**Review by LukasPaczos — APPROVED** [review]

**Comment by dzinad** [comment]
> Can one of help me get an approval on this today as it's nearly impossible for me to rebase this branch daily?

:(

image

**Pull request merged**